DEVELOP.md

```diff
@@ -9,65 +9,66 @@ Dependencies:
 4. [RabbitMQ](http://www.rabbitmq.com/) for workers
 
 
-### Setting up CycleDash
+### 1. Install Dependencies
+
+```bash
+brew install Python                #brew will install the latest Python 2.7.9, which includes pip
+brew install postgres				
+brew install rabbitmq				
+pip install virtualenv
+```
+
+
+### 2. Set up CycleDash
 
 ```bash
 virtualenv venv                    # Initialize a new virtual environment.
-source venv/bin/activate           # Activate your virtual environment.
+source venv/bin/activate           # Activate your virtual environment. This is the only command you will need to run every time you start the app.
 pip install -r requirements.txt    # Install python requirements into virtualenv
-npm install                        # Install JS requirements
+npm install                        # Install JS requirements and packages in package.json
 make initenv                       # Initialize environment file.
-$EDITOR ENV.sh                     # Fill in values.
+$EDITOR ENV.sh                     # Fill in values. (see next step)
 ```
 
 
-### Database
+### 3. Edit the ENV.sh file (generated by `make initenv`)
 
-We use PostgreSQL as our datastore, with the schema described in `schema.sql`.
-
-You can load the schema with `psql cycledash < schema.sql`, and you'll be good
-to go.
-
-On OS X, setting up and running psql might look like this:
+Environment variables which must be exported--edit them in the ENV.sh file make
```

Comment:
@danvk Would something like this make sense: "ENV.sh is generated by 'make initenv' and contains environment variables that need to be exported."

Not sure if the "contains environment variables that need to be exported" bit makes sense?